src/quantum/plugin/QuantumOptions.ts

```diff
@@ -222,7 +222,10 @@ export class QuantumOptions {
         return this.optsTarget === "electron";
     }
     public isTargetUniveral() {
-        return this.optsTarget === "universal";
+        return this.optsTarget === "universal" || this.optsTarget === "npm";

```

Comment:
When `target: "universal"`, `isTargetNpm` evaluates to `false`.

The statement `require` will be used when and only when `isTargetNpm` evaluates to `true`.

Therefore, when `target: "universal"`, the statement used will be `$fsx.s`, like expected